blosc/shuffle.c

```diff
@@ -529,9 +531,11 @@ void unshuffle(size_t bytesoftype, size_t blocksize,
 /* Unshuffle a block.  This can never fail. */
 void unshuffle(size_t bytesoftype, size_t blocksize,
                const uint8_t* _src, uint8_t* _dest) {
-  int unaligned_src = (int)((uintptr_t)_src % 32);
-  int unaligned_dest = (int)((uintptr_t)_dest % 32);
-  int multiple_of_block = (blocksize % (32 * bytesoftype)) == 0;
+  int unaligned_src = (int)((uintptr_t)_src % 16);
```

Comment:
Hmm, I am tempted to use the 'storeu' functions for the SSE2 flavor as well, and then we can completely relax the alignment condition.  But I suppose that if I used the direct assignment in first place is because the difference is speed can be quite important.  But worth another experiment anyway :)